client/src/components/Login/style.scss

```diff
@@ -0,0 +1,269 @@
+.login-container {
+  .container-fluid {
```

Comment:
this styles repeated a lot of times. Maybe the is a reason to move it to general file. how do u think?

Reply:
done